reactjs-todo/e2e/embedded-login.spec.js

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+test('embedded login react', async ({ page }) => {
+  await page.goto('https://localhost:8443/', { waitUntil: 'networkidle' });
```

Comment:
same as previous

Reply:
set to default and worked(load)

Reply:
I set the 'waitUntil' option to 'load' which is the default option and worked as expected. Since more than one comment with the same question marking it to 'resolved'